ReadMe.md

```diff
@@ -9,6 +9,10 @@
 [![Stars](https://img.shields.io/github/stars/praveenscience/Internship-LMS-FrontEnd?style=social)](https://github.com/praveenscience/Internship-LMS-FrontEnd) 
 [![Watchers](https://img.shields.io/github/watchers/praveenscience/Internship-LMS-FrontEnd?style=social)](https://github.com/praveenscience/Internship-LMS-FrontEnd)
 
+Created by the developers, for the developers and the enterprises, **DevTernShip** aims to automate a process that is usually beset by difficulties; The recruitment phase is riddled with disorientation and this makes it difficult for the onboarding interns to make heads or tails of the system.
+
+Here's where **DevTernShip** jumps into the picture - *An Internship Onboarding platform with an innovative learning management system and intuitive performance tools*, we ensure that all is 'smooth sailing' when it comes to the talent pipeline. Moreover, Augmented efficiency is brought about with a simple and robust UI, thus rendering the experience even more dynamic.
```

Comment:
We need the trailer link. Is it possible to embed YouTube or just use the image or something?

Reply:
I added the image.

Reply:
Any possibility of embedding videos? Did you check them out?

Reply:
I Inserted a youtube clickable image to markdown. And Yes, I did check out those - converting the video to a 
GIF...Would that do?

Reply:
Haha, it won't work for a 1 hour video dude. 🤣

Reply:
:sweat_smile: